apps/studio/src/features/editing-experience/components/DiscardChangesModal/DiscardChangesModal.tsx

```diff
@@ -25,9 +25,10 @@ export const DiscardChangesModal = ({
     <Modal isOpen={isOpen} onClose={onClose}>
       <ModalOverlay />
       <ModalContent>
-        <ModalHeader>
+        <ModalHeader maxW="90%">
```

Comment:
maybe a padding-right to the icon width + some would be more "correct". If not on small screens the 10% might still still overlap.

Reply:
Ah right, fixed in b4a9cc906d318474bb8fdb3f133ee99a6b16328d.